### ISSUE TITLE
Simple Pie Gauge Chart Example

### ISSUE THREAD

**Issue by envision** [viserjs/viser#330]
Could you please add an example of how to produce this G2 demo with Visor?

https://antv.alipay.com/zh-cn/g2/3.x/demo/pie/pie7.html

PS. I'm experienced with dataviz but AntV and Visor is a new comer. English documentation is needing more work.

**Comment by huxiaoyun** [comment]
pie  https://viserjs.github.io/demo.html#/viser/pie/basic-pie

gauge   https://viserjs.github.io/demo.html#/viser/gauge/basic-gauge


label pie  https://viserjs.github.io/demo.html#/viser/others/custom-label-pie

complex lable pie  https://viserjs.github.io/demo.html#/viser/others/daily-life-visualization

**Comment by envision** [comment]
Thanks @huxiaoyun, I'm aware of these examples. However as I'm unfamiliar with G2 still at this point, I couldn't figure out how to configure Visor based on these demos to match that simpler "radial bar chart" demo that was done with bare G2.

My requirement is to produce similar radial bar chart "gauge" as seen in that G2 demo I linked. In addition, I need it to be a solid fill color, taken from a gradient at the location of the percentage value. And this color to be animated from the gradient's starting color until the color at the percentage value.

I could produce this kind of chart with D3 directly, but the reason I turned to your solution was to save time...apparently I would need to take a decent time off to learn G2.
